packages/widgets/src/FieldSetField/FieldSetField.js

```diff
@@ -0,0 +1,80 @@
+import propTypes from '@dhis2/prop-types'
+import React from 'react'
+
+import { sharedPropTypes } from '@dhis2/ui-constants'
+import { Field, FieldSet, Legend } from '@dhis2/ui-core'
+
+/**
+ * @module
+ * @param {FieldSetField.PropTypes} props
+ * @returns {React.Component}
+ *
+ * @example import { FieldSetField } from '@dhis2/ui'
+ *
+ * @example import { FieldSetField } from '@dhis2/ui-widgets'
+ */
+const FieldSetField = ({
+    children,
+    className,
+    disabled,
+    helpText,
+    validationText,
+    label,
+    required,
+    dataTest,
+    valid,
+    error,
+    warning,
+}) => (
+    <FieldSet classname={className} dataTest={dataTest}>
+        <Field
+            label={label}
+            disabled={disabled}
+            required={required}
+            helpText={helpText}
+            validationText={validationText}
+            error={error}
+            warning={warning}
+            valid={valid}
+        >
+            {children}
+        </Field>
+    </FieldSet>
+)
+
+FieldSetField.defaultProps = {
+    dataTest: 'dhis2-uicore-togglegroupfield',
```

Comment:
`dataTest: 'dhis2-uicore-togglegroupfield'` -> `dataTest: 'dhis2-uiwidgets-fieldsetfield'`

Maybe even ui-fieldsetfield, but it's probably better to make an issue and leave it for a later patch.

Reply:
Thanks, there may be more of those. I'll do a search.

Reply:
1c2cfa9